pkg/cmd/admin/policy/who_can.go

```diff
@@ -90,8 +102,14 @@ func (o *whoCanOptions) run() error {
 	} else {
 		fmt.Printf("Namespace: %s\n", resourceAccessReviewResponse.Namespace)
 	}
+
+	resourceDisplay := o.resource.Resource
+	if len(o.resource.Group) > 0 {
+		resourceDisplay = resourceDisplay + "." + o.resource.Group
```

Comment:
I have been conditioned to cringe at the sight of string concats but hey...c'est la vie!